logger/logger.go

```diff
@@ -81,23 +81,67 @@ func GetLogger(isDebug bool, prefix string) *Logger {
 	}
 }
 
+// GetSecondaryPrefix returns the first secondary prefix
 func (l *Logger) GetSecondaryPrefix() string {
-	return l.secondaryPrefix
+	if len(l.secondaryPrefixes) == 0 {
+		return ""
+	}
+	return l.secondaryPrefixes[0]
 }
 
+// UpdateSecondaryPrefix replaces all secondary prefixes with the new one
+// for backward compatibility
 func (l *Logger) UpdateSecondaryPrefix(prefix string) {
```

Comment:
We can remove backward compatibility! Not worth the complexity, just make sure we upgrade all tester repos that currently use this.